carma_utils/include/carma_utils/containers/containers.h

```diff
@@ -31,10 +31,13 @@ namespace containers
  * 
  * \param input The input vector to downsample
  * \param n The count of the elements to save
+ * \param include_last_point If true, append the last point
```

Comment:
Same comments apply here (regarding the `included_last_point` variable and the function description comments. Though I just realized that we can completely remove this ROS 1 carma_utils/containers, as it is unused anywhere in our system now. We can handle this removal in a future ROS 2 migration cleanup story.

Reply:
I have added `has_included_last_point` variable, so this one should be good now